.travis.yml

```diff
@@ -3,11 +3,14 @@ rust:
   - stable
   - 1.38.0  # lowest rust release against which we guarantee compatibility.
 cache: cargo
+env:
+  - FEATURES='--features chrono'
+  - FEATURES=''
 before_install:
   - rustup component add clippy
   - rustup component add rustfmt
 script:
-  - cargo build --verbose
+  - cargo build --verbose $FEATURES
   - cargo test --verbose
   - cargo clippy --verbose
-  - cargo fmt && [ `git status -s | wc -l` -eq 0 ] # check that all files are formatted with `cargo fmt`
+  - cargo fmt && [ `git status -s | wc -l` -eq 0 ] # check that all files are formatted with `cargo fmt`
```

Comment:
nit-picking: this is removing the final linebreak and wouldn't have been necessary... but we can keep it as-is, doesn't hurt, just doesn't look 100% nice.